src/main/java/org/spongepowered/api/command/spec/CommandSpec.java

```diff
@@ -354,6 +354,15 @@ public boolean testPermission(CommandSource source) {
     }
 
     /**
+     * Gets the extended description used with this command if any is present
+     *
+     * @return the extended description.
+     */
+    public Optional<Text> getExtendedDescription(CommandSource source) {
```

Comment:
`CommandSpec` is a final class that will always return the field, so the parameter is really not necessary here.